### ISSUE TITLE
Build failed - likely .readthedocs.yml ignored, which is in branch, but not in master

### ISSUE THREAD

**Issue by NumesSanguis** [readthedocs/readthedocs.org#7329]
## Details

* Read the Docs project URL: https://facsvatar.readthedocs.io/en/latest/index.html
* Build URL (if applicable): https://readthedocs.org/projects/facsvatar/builds/11544992/
* Read the Docs username (if applicable): NumesSanguis
* GitHub repo branch v0.4.0: https://github.com/NumesSanguis/FACSvatar/tree/v0.4.0

## Expected Result

Successfully build v0.4.0 branch of my GitHub project, using `.readthedocs.yml` for building the docs.

### Build failed:
Working:
```
git clone --no-single-branch --depth 50 https://github.com/NumesSanguis/FACSvatar.git .
git checkout --force origin/v0.4.0
git clean -d -f -f
git submodule sync
git submodule update --init --force --recursive addons/FACSvatar-Blender
python2.7 -mvirtualenv  /home/docs/checkouts/readthedocs.org/user_builds/facsvatar/envs/v0.4.0
/home/docs/checkouts/readthedocs.org/user_builds/facsvatar/envs/v0.4.0/bin/python -m pip install --upgrade --no-cache-dir pip
/home/docs/checkouts/readthedocs.org/user_builds/facsvatar/envs/v0.4.0/bin/python -m pip install --upgrade --no-cache-dir Pygments==2.3.1 setuptools==41.0.1 docutils==0.14 mock==1.0.1 pillow==5.4.1 alabaster>=0.7,<0.8,!=0.7.5 commonmark==0.8.1 recommonmark==0.5.0 sphinx<2 sphinx-rtd-theme<0.5 readthedocs-sphinx-ext<1.1
cat docs/conf.py
```

Failed:
```python
$ python /home/docs/checkouts/readthedocs.org/user_builds/facsvatar/envs/v0.4.0/bin/sphinx-build -T -b readthedocs -d _build/doctrees-readthedocs -D language=en . _build/html
Running Sphinx v1.8.5
loading translations [en]... done

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/facsvatar/envs/v0.4.0/lib/python2.7/site-packages/sphinx/cmd/build.py", line 303, in build_main
    args.tags, args.verbosity, args.jobs, args.keep_going)
  File "/home/docs/checkouts/readthedocs.org/user_builds/facsvatar/envs/v0.4.0/lib/python2.7/site-packages/sphinx/application.py", line 228, in __init__
    self.setup_extension(extension)
  File "/home/docs/checkouts/readthedocs.org/user_builds/facsvatar/envs/v0.4.0/lib/python2.7/site-packages/sphinx/application.py", line 449, in setup_extension
    self.registry.load_extension(self, extname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/facsvatar/envs/v0.4.0/lib/python2.7/site-packages/sphinx/registry.py", line 475, in load_extension
    raise ExtensionError(__('Could not import extension %s') % extname, err)
ExtensionError: Could not import extension m2r2 (exception: No module named m2r2)

Extension error:
Could not import extension m2r2 (exception: No module named m2r2)
```

As the output says it's using Python 2.7, it seems that the `.readthedocs.yml` in the repo is being ignored. In that yml, it states `python=3.7`. Also in this yml I have pip install m2r2.

## Other
I have a warning saying:
" Your project, FACSvatar, needs to be reconfigured in order to continue building automatically after January 31st, 2019. For more information, see our documentation on webhook integrations. "
However, it seems to work fine, as pushes trigger automatic building. The documentation is also not clear in how to make this warning go away.

**Comment by stsewd** [comment]
>  However, it seems to work fine, as pushes trigger automatic building. The documentation is also not clear in how to make this warning go away.

You need to click on the link in order to make it disappear. Our templates are being reworked and that problem is addressed there.

About your other problem, you are using a conda environment file https://github.com/NumesSanguis/FACSvatar/blob/5fd2c7c16019c528fdf12082f8d7502a4788d2ea/.readthedocs.yml, not a Read the Docs configuration file, please check https://docs.readthedocs.io/en/stable/config-file/v2.html

**Comment by NumesSanguis** [comment]
> You need to click on the link in order to make it disappear. Our templates are being reworked and that problem is addressed there.

I'm not sure to which link you are referring (I only saw a link to the documentation), but the issue is gone now. I had 2 hooks instead of one, so I thought this was the issue. I deleted it, but that the warning did not go away (instantly).
However, today the warning message was suddenly not there any more.

Thank you for the link on how to make a RtD configuration .yml (I wrongly assumed it to be the same as for conda). I updated the file, and it build successfully.
